docs/endpoints.md

```diff
@@ -1,16 +1,18 @@
-### LLM Backends
-
 You can use MemGPT with various LLM backends, including the OpenAI API, Azure OpenAI, and various local (or self-hosted) LLM backends.
 
-#### OpenAI
+## OpenAI
 To use MemGPT with an OpenAI API key, simply set the `OPENAI_API_KEY` variable:
 ```sh
 export OPENAI_API_KEY=YOUR_API_KEY # on Linux/Mac
 set OPENAI_API_KEY=YOUR_API_KEY # on Windows
 $Env:OPENAI_API_KEY = "YOUR_API_KEY" # on Windows (PowerShell)
 ```
 
-#### Azure
+#### OpenAI Proxies
```

Comment:
Did you mean to make this a `##` heading?

Reply:
I thought it was more clear as a subsection of OpenAI (since a custom endpoint is different)

Reply:
but we can change in the future